app/models/solidus_bolt/payment_source.rb

```diff
@@ -4,6 +4,5 @@
 
 module SolidusBolt
   class PaymentSource < SolidusSupport.payment_source_parent_class
-    belongs_to :user, class_name: 'Spree::User', optional: true
```

Comment:
What do you think to add a payment method validator In order to prevent that we can create a PaymentSource without the PaymentMethod

Reply:
fixed in https://github.com/nebulab/solidus_bolt/pull/89/commits/d511d4222ae067c87c9ef303d220e1f64ca0e146